Thousand Sons Legion Units and WLT

Arkangelmark5 opened the pull request:
Thousand Sons Legion Units and WLT

## Added Units
Magnus the Red
Sekhmet
Contemptor-Orison
Castellax-Achea
Khenetai
Ammitara Occult

## Fixed Units
Ahriman
Amon

##Added WLT
Evoker of Pain
Magister of Prospero
Eidolon of Suffering

##Test Case
![image](https://user-images.githubusercontent.com/53609638/183078327-e328eac1-e8d3-4e80-9fa6-6b5339689e69.png)
![image](https://user-images.githubusercontent.com/53609638/183078386-6f360312-7ffb-4f71-b095-6f83cc7ec6cb.png)
![image](https://user-images.githubusercontent.com/53609638/183078424-a752fbf1-2f30-401b-9371-f0c6d1ba421e.png)
![image](https://user-images.githubusercontent.com/53609638/183078490-c4fbd832-9683-4466-ad9e-d05f590752ec.png)
![image](https://user-images.githubusercontent.com/53609638/183078599-1d8e195e-5852-4e55-a83d-5faf024abe1e.png)
![image](https://user-images.githubusercontent.com/53609638/183078694-b61e69e1-ce68-41fa-b709-3a286e59c556.png)
![image](https://user-images.githubusercontent.com/53609638/183078748-c17c958a-b30b-4b1e-aa6b-491e4ecbac44.png)
![image](https://user-images.githubusercontent.com/53609638/183078787-e823f761-aef9-4f89-90ae-9b62fcd83bb4.png)

UrsinePatriarch reviewed (changes requested):
Please de-nest Master of the Legion as we discussed.
What's the "*" in Amon's Reliquary for? 
The Osiron Magus can only be taken by itself, I believe.
If an Osiron takes a 2nd force blade, do you have a second way of checking its in-built weapon?
Do Khenetai have any other options for their Force Sword slot? If not, it can just be written as an SE "Pair of Achea Force Swords" that includes 2 links in it rather than needing to select it twice for every model added.
Do you have the Khenetai Blademaster's Artificer Armour set to actually change his Save in his profile if selected?
Evoker of Pain seems to be formatted incorrectly on multiple lines; did you copy and paste it from somewhere?

Arkangelmark5 commented:
Fixed formatting with the Evoker of Pain
Fixed Khenetai Blademaster's Artificer Armour to effect his save instead of his strength
Redid the Khenetai force swords as a Pair of Force Swords SE and added the Force Sword rules to it.
Clarified Orison Magus as a upgrade rather then the dreadnought
Moved Orison Dreadnought Combi weapon, so It's no longer nested under the force blade.
Clarified Amons Weapons to have a Melee and Shooting version
